google-cloud-spanner-hibernate-testing/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>google-cloud-spanner-hibernate</artifactId>
+		<groupId>com.google.cloud</groupId>
+		<version>0.1.0-SNAPSHOT</version>
+	</parent>
+
+	<properties>
+		<checkstyle.skip>true</checkstyle.skip>
+	</properties>
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>integration-testing</artifactId>
+	<name>Google Cloud Spanner Hibernate Integration Testing</name>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.hibernate</groupId>
+			<artifactId>hibernate-testing</artifactId>
+			<version>5.4.0.Final</version>
+			<scope>test</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>com.google.cloud</groupId>
+			<artifactId>google-cloud-spanner-hibernate-dialect</artifactId>
+			<version>0.1.0-SNAPSHOT</version>
+		</dependency>
+
+		<dependency>
```

Comment:
This dependency should come from `google-cloud-spanner-hibernate-dialect`.

Reply:
Does this comment refer to the Simba Driver? In this case I kept the Simba driver with the tests as it is only used in an integration test situation.